R/grid.draw_ggsurvfit.R

```diff
@@ -0,0 +1,34 @@
+#' Draw ggsurvfit object
+#'
+#' `grid::grid.draw()` methods for objects of classes 'ggsurvfit' and 'ggcuminc'.
+#' These are implemented to allow users to directly call `ggplot2::ggsave()`
```

Comment:
Could we add the reason why they could not use it without grid.draw?

Reply:
if only I understood all the details! 😆

Reply:
Could we at least write high-level why we need a function to do this and why we could not use ggsave without drawing the object.

Reply:
I really do not know the details